lib/sen_tweet_web/telemetry.ex

```diff
@@ -32,7 +43,29 @@ defmodule SenTweetWeb.Telemetry do
       summary("vm.memory.total", unit: {:byte, :kilobyte}),
       summary("vm.total_run_queue_lengths.total"),
       summary("vm.total_run_queue_lengths.cpu"),
-      summary("vm.total_run_queue_lengths.io")
+      summary("vm.total_run_queue_lengths.io"),
+
+      # Bitfeels Pipeline Source Metrics
+      counter("bitfeels.pipeline.source.id", tags: [:track]),
+
+      # Bitfeels Pipeline Dispatcher Metrics
+      last_value("bitfeels.pipeline.dispatcher.enqueue.queue_length"),
+      last_value("bitfeels.pipeline.dispatcher.enqueue.pending_demand"),
+
+      last_value("bitfeels.pipeline.dispatcher.dequeue.queue_length"),
+      last_value("bitfeels.pipeline.dispatcher.dequeue.demand"),
+      last_value("bitfeels.pipeline.dispatcher.dequeue.events"),
+
+      # Bitfeels Pipeline Sentiment Metrics
+      last_value("bitfeels.pipeline.sentiment.number_of_events"),
+      counter("bitfeels.pipeline.sentiment.id", tags: [:track]),
+      summary("bitfeels.pipeline.sentiment.score", tags: [:track]),
+
+      # Bitfeels Tweet Parser Metrics
+      counter("bitfeels.tweet.parser.id", tags: [:tweet_type]),
+
+      # Bitfeels Tweet Sentiment Metrics
+      summary("bitfeels.tweet.sentiment.senpy_response_time", unit: {:microsecond, :millisecond})
```

Comment:
These are the built-in aggregation functions that we can apply to our events. Each one of these functions apply to a measurement in our event. For example, here the event is `bitfeels.tweet.sentiment` and the measurement is `senpy_response_time`

Feel free to mess around with these. There's a chart for each one of these. Look in the module doc for a link to documentation.